### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by fahimy143**
### Project Requirements 

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+/*eslint-disable */
+class Calculator extends Component {
```
Comment (julie-ify) [inline src/components/Calculator.js]: - [ ] Kindly enable `eslint`. You can disable the rule for the `prefer-stateless-function` but do not disable all eslint rules.

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```
Comment (julie-ify) [inline src/App.js]: - [ ] Kindly change the `App` component to a class-based component.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  render() {
+    return (
+      <div className="container">
+        <div className="wrapper">
```
Comment (julie-ify) [inline src/components/Calculator.js:7]: - [ ] Kindly match the given design and place the `%` and `+/-`  buttons accordingly as seen from the design.

Your Implementation | Design guideline
:---:|:---:
 |

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @fahimy143,

Good job so far!

## Highlights!
- GitFlow ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization and working app ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review and here._

- [x] Kindly format your README to be well documented using [the Microverse template](https://github.com/microverseinc/readme-template) instead of having the README generated from `create react app`.

- [x] Kindly remove all the boilerplate that got created by the CRA (texts, images, and styles), check the public folder.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @fahimy143  👍🏽, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**